README.md

```diff
@@ -45,25 +44,88 @@ Create an input configuration object and add a redux-inputs reducer to your redu
         inputs: createInputsReducer(inputConfig)
     });
 
-Note that `inputs` is the default mount point for redux-inputs state. You can
- configure this in your `inputConfig` object.
+### Step 4 - Connect
```

Comment:
I think this step's example needs some work, it's not very clear how it all works. So does `connectWithInputs` return a function similar to `connect`? Does a `mapStateToProps` and `mapDispatchToProps` fit in somewhere?